ssl/ssl_lib.c

```diff
@@ -675,6 +679,11 @@ int SSL_is_dtls(const SSL *s)
     return SSL_IS_DTLS(s) ? 1 : 0;
 }
 
+int SSL_is_tls13_plus(const SSL *s)
```

Comment:
This function is only tangentially relevant to this PR so, as it is contentious, I have removed it for now and inserted a TODO asking the question about how applications determine whether they should call `SSL_key_update()` or `SSL_renegotiate()`. To be dealt with in a future PR.